src/Arcus.Observability.Tests.Integration/Serilog/ApplicationInsightsSinkTests.cs

```diff
@@ -143,6 +143,27 @@ public void LogDependency_SinksToApplicationInsights_ResultsInDependencyTelemetr
             }
         }
 
+        [Fact]
+        public void LogServiceBusDependency_SinksToApplicationInsights_ResultsInServiceBusDependencyTelemetry()
+        {
+            // Arrange
+            using (ILoggerFactory loggerFactory = CreateLoggerFactory())
+            {
+                ILogger logger = loggerFactory.CreateLogger<ApplicationInsightsSinkTests>();
+
+                string entityName = _bogusGenerator.Commerce.Product();
+                bool isSuccessful = _bogusGenerator.PickRandom(true, false);
+                TimeSpan duration = _bogusGenerator.Date.Timespan();
+                Dictionary<string, object> telemetryContext = CreateTestTelemetryContext();
+
+                // Act
+                logger.LogServiceBusDependency(entityName, isSuccessful, DateTimeOffset.UtcNow, duration, ServiceBusEntityType.Queue, telemetryContext);
```

Comment:
This is not showing up in Application Insights; at least not correctly

Reply:
I think it shows up now correctly.

Reply:
I'm still seeing the old `Azure Resource`, did you see anything else?
![image](https://user-images.githubusercontent.com/4345663/79320534-41e2bb00-7f0a-11ea-9804-076996a461f6.png)

Also not that there are properties that should be gone

Reply:
![image](https://user-images.githubusercontent.com/9039753/79320668-76567700-7f0a-11ea-9f59-b4c820d8d1d4.png)

Reply:
:+1: Can you removes these 2 properties as well along wiht `DependencyStartTime` & `DependencyDuration` please?

Also, why does it say "EventDescription"? Because it's a dependency?

Reply:
That's the name we use for the context on all log types.

Reply:
Those properties are by default added to custom dependencies too. Just reusing functionality here.

Reply:
I think we should get rid of them as the information is already there and more noise making it harder to find what you need

Reply:
Can be via a dedicated PR if you want, but don't forget to create an issue then.

Reply:
In both the custom dependency as in the service bus dependency?

Reply:
Yes because we just use them to pass it to the converter so that he can set the correct properties unless I'm missing something ofcourse.

Reply:
Addd an issue for this: https://github.com/arcus-azure/arcus.observability/issues/81